platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -2,14 +2,14 @@
 
 Inspired by the cross-functional cooperation promised by DevOps, platform
 engineering has begun to emerge in enterprises as an explicit form of that
-cooperation: platform teams curate and present foundational capabilities,
-frameworks and experiences to facilitate and accelerate the work of internal
-customers such as application developers, data scientists and information
-workers.
+cooperation, particularly for cloud-native application development and delivery.
+Platforms curate and present foundational capabilities, frameworks and
+experiences to facilitate and accelerate the work of internal customers such as
+application developers, data scientists and information workers.
```

Comment:
IMO `, particularly for cloud-native application development and delivery.` feels stuck into this sentence.

I wonder if we can create a separate sentence for this? One suggestion would be to replace this with a sentence either inside or at the end of this paragraph.

Here is a small attempt to try and show what I mean:

```suggestion
cooperation. Platforms curate and present foundational capabilities, frameworks and
experiences to facilitate and accelerate the work of internal customers such as
application developers, data scientists and information workers. Cloud-native application development and delivery has helped set a new bar for speed to market, and delivering the tools to support this requires the same level of focus. 
```

Reply:
Makes sense. I'm rewriting the "cloud" bit as another sentence rather than an aside in the first sentence.